test/WebSites/FilesWebSite/Controllers/UploadFilesController.cs

```diff
@@ -21,5 +23,22 @@ public class UploadFilesController : Controller
 
             return Json(resultUser);
         }
+
+        [HttpPost("UploadProductSpecs")]
+        public IActionResult ProductSpecs(Product product)
+        {
+            if (!ModelState.IsValid)
+            {
+                return BadRequest(ModelState);
+            }
+
+            var files = new Dictionary<string, List<string>>();
```

Comment:
> Deserves a comment about returning filenames instead of IFormFiles.

I think what we currently have is fine. This shows that we are able to model bind and get values out of form files.

> You could also create another public DTO and use that here since everything is accessible w/in FormFileUploadTest.

I don't think this is recommended (even though we are doing in-memory approach)...for example, at one point @rynowak was trying to run functional tests with real servers in which case we do not want the tight coupling